assets/js/role_slider.js

```diff
@@ -0,0 +1,12 @@
+// const roleSliderContainer = document.getElementById("roleSlider");
+// const roleSliderTabs = document.querySelectorAll(".tab");
+// roleSliderTabs[0].children[0].classList.add('selected');
+
+// roleSliderContainer.addEventListener('mousedown', (e) => {
+//     roleSliderTabs.forEach(tab => {
+//         tab.children[0].classList.remove('selected');
+//         if (tab.contains(e.target)) {
+//             tab.children[0].classList.add('selected');
+//         };
+//     });
+// });
```

Comment:
Weird question – I thought we removed this?

Reply:
lol omg my rebasing was acting up on this branch. I'll remove it in a sec 😆